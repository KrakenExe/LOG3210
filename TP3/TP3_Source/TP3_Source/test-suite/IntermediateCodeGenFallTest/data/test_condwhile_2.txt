Declaration
num a;
num b;
bool c;
EndDeclaration

while(1 >= a) {
  a = 42;
  b = 1 / 3 + 5;
}
  
