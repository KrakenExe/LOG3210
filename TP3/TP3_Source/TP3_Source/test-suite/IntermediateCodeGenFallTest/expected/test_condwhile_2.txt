_L1
ifFalse 1 >= a goto _L0
a = 42
_L2
_t1 = 1 / 3
_t0 = _t1 + 5
b = _t0
goto _L1
_L0
